calcom/cal.com
# fix: how-to-setup-api-in-a-local-instance.mdx in docs with correction after api-v2

After the implementation of api-v2, some of the documents are not yet updated and could you make developers spend too much time on finding these things. 

## What does this PR do?



- Fixes #XXXX (GitHub issue number)
- Fixes CAL-XXXX (Linear issue number - should be visible at the bottom of the GitHub issue description)
- This is an issue I noticed while going through the docs.
- This changes some text in the documentation for how to setup api in local instance page



## Mandatory Tasks (DO NOT REMOVE)

- [✅] I have self-reviewed the code (A decent size PR without self-review might be rejected).
- [✅] I have updated the developer docs in /docs if this PR makes changes that would require a [documentation change](https://cal.com/docs). If N/A, write N/A here and check the checkbox.
- [N/A] I confirm automated tests are in place that prove my fix is effective or that my feature works.

## How should this be tested?



- Are there environment variables that should be set? No
- What are the minimal test data to have? NA
- What is expected (happy path) to have (input and output)? Check the edited docs page after update under the link https://cal.com/docs/developing/guides/api/how-to-setup-api-in-a-local-instance
- Any other important info that could help to test that PR

## Checklist



- I haven't checked if my changes generate no new warnings
